pyproject.toml

```diff
@@ -63,6 +98,7 @@ select = [
     "E",
     "F",
     "SIM1",
+    "UP034",
```

Comment:
```suggestion
    "UP0",
```

Reply:
Oops, I though I had changed that. But why not just "UP"?

Reply:
> But why not just "UP"?

I think it could be `UP`. I.e., select all `UP` rules.